src/common/engine.js

```diff
@@ -928,7 +928,13 @@ export class CurrencyEngine<T> {
     return dataDump
   }
 
-  makeSpend(edgeSpendInfo: EdgeSpendInfo): Object {
+  makeSpendCheck(edgeSpendInfo: EdgeSpendInfo): {
+    edgeSpendInfo: EdgeSpendInfo,
+    nativeBalance: string,
+    currencyCode: string,
+    denom: EdgeDenomination,
+    skipChecks: boolean
```

Comment:
I think we shouldn't return `skipChecks` here because it is not being mutated by `makeSpendCheck`.

Reply:
That wasn't my change, but I do think there's value since `makeSpendCheck` sort of mutates it. It defaults to `false` if undefined. We'd have to mirror that change in each makeSpend otherwise.